source/components/ocean_component.cpp

```diff
@@ -68,20 +68,12 @@ void OceanComponent::init( Core* coreptr ) {
     oceanflux_constrain.name = "atm_ocean_constrain";
 	
     Tgav.set( 0.0, U_DEGC );
-	heatflux.set( 0.0, U_W_M2 );
-        k_max.set( 0.8, U_W_M2_K );    // default
-        t_mid.set( 2.75, U_K );        // default
-        k_min.set( 0.0, U_W_M2_K );   // default
-        slope.set( -3.0, U_1_K );      // default
-        
-        m_min_k_so_far = std::numeric_limits<double>::max();
 
 	lastflux_annualized.set( 0.0, U_PGC );
         
     // Register the data we can provide
     core->registerCapability( D_OCEAN_CFLUX, getComponentName() );
 	core->registerCapability( D_OCEAN_C, getComponentName() );
-    core->registerCapability( D_HEAT_FLUX, getComponentName() );
```

Comment:
I take it these heat flux capabilities have been moved to the DOEClim component?

Edit: Actually, it doesn't look like they have.  Are we just not supporting retrieving the ocean heat flux anymore, or is it going to known by a different name going forward?  Apart from being a serious backward-incompatible change, it would be kind of unfortunate if we could no longer get this information from the model.

Reply:
With DOECLIM, we calculate a flux into the mixed layer and a flux into the interior. For backward compatibility, I'll make a note to also do a total heat flux.

Reply:
Heat flux is calculated in the DOECLIM component now.  @bvegawe that's a good idea to have total heat flux into the ocean.

Reply:
done